content/docs/get-started-with-neon/using-api-keys.md

```diff
@@ -8,7 +8,7 @@ A Neon user is identified by their email address.
 
 A user registers and authenticates in Neon Web UI with their GitHub or Google account. More authentication methods are coming soon.
 
-Once authenticated, a user can create and access [Projects](#project) and [query Project data](../tutorials#query-via-ui). You can also manage [Postgres Users](#postgres-users) and [Databases](#postgres-databases) in each Project.
```

Comment:
projects and users were supposed to link to the sections that are currenly in this doc:
https://websitemain62807-redirects.gtsb.io/docs/cloud/concepts/

Reply:
Projects doesn't seem to have been rewritten or moved yet. If the information is still up to date, I can move the project link (and the Postgres users) to the glossary and add those links accordingly.

Reply:
Projects look up to date.

Reply:
I've added Project to the glossary